android/app/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'realm-android'
 
 project.ext.envConfigFiles = [
     debug: ".env.bt",
+    staging: ".env.bt.staging",
```

Comment:
This won't compile I think, more changes are needed to add a new build variant
I can make those changes here

Reply:
@juancruzgs thanks!